WordPressCom-Stats-iOS/UI/InsightsTableViewController.m

```diff
@@ -378,7 +379,7 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
                         WPStatsViewController *statsViewController = (WPStatsViewController *)self.navigationController;
                         [self.statsDelegate statsViewController:statsViewController openURL:action.url];
                     } else {
-                        [[UIApplication sharedApplication] openURL:action.url];
+                        [ExtensionUtils openURL:action.url fromController:self];
```

Comment:
The nice thing is that `ExtensionUtils` kind of explains what this is for (ie compatibility with app extensions). 100% up to you, but if it were even more on-the-nose with its name maybe my comment… comment above would be irrelevant. Like "AppExtensionUtils"?

Just a thought. Don't let this comment stop you from merging if you hate the idea.